### PR TITLE
allow sorting by 'modified_on'

### PR DESCRIPTION
**Pull request by beav**

### CODE REVIEW

File: tests/test_v1_api.py [hunk 1]
```diff
@@ -173,6 +173,22 @@ def test_fetch_baseline_list_sort(self):
         # check that the ascending result is the inverse of the descending result
         self.assertEqual(desc_result["data"][::-1], asc_result["data"])
 
+    def test_fetch_baseline_list_sort_updated_on(self):
+        response = self.client.get(
+            "api/system-baseline/v1/baselines?order_by=updated_on",
```
Comment (ehelms) [inline tests/test_v1_api.py]: I see updated_on here, and then modified_on mentioned elsewhere

Comment (beav) [reply tests/test_v1_api.py]: oops, typo.

probably not great that this passes tests, will make the test stricter.

**Comment by beav** [comment]
@ehelms updated so the test actually fails if you typo the sort name.

I'm having some trouble getting openapi to validate query params which is why we were able to send updated_on. I will look into this outside of this PR.

**Comment by ehelms** [comment]
We have a bit of a disconnect. The API response gives us `updated` as a field and thus I would expect to sort on the same name.

```
account: ""
created: "2019-07-25T13:46:39.932516Z"
display_name: "foo"
fact_count: 5
id: "46501a82-f1e9-4c0f-9e7f-ba27282ac18b"
updated: "2019-07-26T13:59:04.911270Z"
```

**Pull request merged**